### PR TITLE
UA custom element example

### PR DESCRIPTION
**Pull request by ahy3nz**
#7 #9 
Deals with united-atom or coarse-grained representations in mBuild and Foyer, performing a simulation with HOOMD, analyzing with Freud

However, this does not cover using the coarse-graining module in mbuild, as that module is somewhat under-developed and un-used

### CODE REVIEW

**Comment by rmatsum836** [comment]
Just went through your notebook example and it works.  This is a good addition to the workflows and starts to give us some diversity in terms of simulation engines.

**Comment by rmatsum836** [comment]
I think this example is good to merge if someone else wants to review it.

File: ua-custom-element/ua_custom_element_hoomd_freud.ipynb [hunk 1]
```diff
@@ -0,0 +1,516 @@
...
+    "\n",
+    "dump = hoomd.dump.gsd('traj.gsd', 1000, group=all, overwrite=True)\n",
+    "hoomd.run(1e7)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After conducing the HOOMD simulation, we will use `freud`, `gsd`, and `numpy` to analyze the simulation, looking at various RDFs"
```
Comment (daico007) [inline ua-custom-element/ua_custom_element_hoomd_freud.ipynb]: typo

**Comment by ahy3nz** [comment]
I added installation instructions, but if [this PR](https://github.com/mosdef-hub/mbuild/pull/622) gets merged, I think the notebook could be simplified down. As for making the simulation longer or plots nicer, those could be nice but I don't think that's super high priority/the point of the notebook, it's just to show how these things would be done

**Comment by uppittu11** [comment]
Well, this is new
image

Does `freud 2.0.0` no longer do exclusions in RDF calculations or something?

This is still good to go, in my view!

**Pull request merged**